lib/modules/nightMode.js

```diff
@@ -137,7 +140,9 @@ module.beforeLoad = () => {
 	toggle.addCLI('ns');
 	if (module.options.nightSwitch.value) toggle.addMenuItem(i18n('nightModeToggleTitle'), 7, '☽', '☀');
 
-	handleAutomaticNightMode();
+	if (module.options.automaticNightMode.value !== 'none') {
```

Comment:
How about `MediaQueryList.addListener("that dark theme css", handleAutomaticNightmode)`

https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryList/addListener